news/12095.bugfix.rst

```diff
@@ -0,0 +1 @@
+Consistently report whether a dependency comes from an extra.
```

Comment:
As mentioned in the PR description I did a second minor bugfix for which no issue exists. Do I understand correctly there is no way to add a second bugfix entry referring to this same PR number? Should I create an issue just to be able to refer to it?

Reply:
I would move the other fragment to `11924.feature.rst` and name this a bugfix instead. It doesn’t feel right to put _Improve_ under the bug fix section.

Reply:
Thanks, I wasn't sure which of the two was most appropriate.